properties/properties.go

```diff
@@ -45,7 +39,7 @@ func fetchQuiteLiteSource(alternateURIFlag string) (altsrc.InputSourceContext, e
 
 	}()
 	ret := QuiteliteInputSource{}
-	err = json.NewDecoder(resp.Body).Decode(&ret)
+	err = json.NewDecoder(resp.Body).Decode(&ret.valueMap)
```

Comment:
is this the main bug?

Reply:
There were a few bugs:
1. This one
2. There was a problem with conversions of floats / ints.
3. The actual function wasn't being called (https://github.com/Netflix/titus-executor/pull/161/files#diff-7cae71039937aae66ba05a7cdcf970fcR75)
4. The flags needed to be enabled with altsrc. https://github.com/Netflix/titus-executor/pull/161/files#diff-587d2988572f886f6e2554c7e0ea6ba7R229